config/momento_proxy.toml

```diff
@@ -32,6 +32,9 @@ default_ttl = 900
 # memory_cache_ttl_seconds = 0
 # the protocol can be "memcache" or "resp" (Redis), the default is memcache
 # protocol = "memcache"
+# Buffer sizes for read/write on the connection from the client application to the proxy
+# Defaults to 16 KiB. Can be tuned for performance based on item size distribution for the cache.
+# buffer_size = 16384
```

Comment:
Should we remark that the buffer size is rounded to the nearest page size under the hood?

Reply:
Done in https://github.com/momentohq/momento-proxy/pull/43/commits/619131542796add1be5c9f64b4fc5ed4f7fa49cf